packages/@okta/i18n/src/properties/login_pl.properties

```diff
@@ -556,6 +556,22 @@ password.complexity.no_username = \u017Cadnej cz\u0119\u015Bci nazwy u\u017Cytko
 password.complexity.no_first_name = nie mo\u017Ce zawiera\u0107 imienia
 password.complexity.no_last_name = nie mo\u017Ce zawiera\u0107 nazwiska
 
+# Requirements as HTML list.
+# These are copy of existing strings, but we need them since in a list, the starting letter has to be in CAPS and no period at the end.
+password.complexity.requirements.header = Wymagania dotycz\u0105ce has\u0142a:
+password.complexity.length.description = co najmniej nast\u0119puj\u0105c\u0105 liczb\u0119 znak\u00F3w: {0}
```

Comment:
Thought this was weird, but turns out it's okay!

<img width="1293" alt="Screen Shot 2020-06-08 at 10 52 53 AM" src="https://user-images.githubusercontent.com/24683447/84063656-38a82480-a976-11ea-9136-7d6946a9318a.png">